pkg/controller/node/nodecontroller.go

```diff
@@ -464,6 +551,13 @@ func (nc *NodeController) monitorNodeStatus() error {
 		nc.knownNodeSet[added[i].Name] = added[i]
 		// When adding new Nodes we need to check if new zone appeared, and if so add new evictor.
 		zone := utilnode.GetZoneKey(added[i])
+
+		// TODO(kevin-wangzefeng): need to identify a proper rate limiter
```

Comment:
To tell the truth I'm not sure we need to rate limit this at all - @smarterclayton @davidopp